retroactive/src/ousd.ts

```diff
@@ -0,0 +1,183 @@
+import fs from "fs";
+import { BigNumber } from "ethers";
+import EthereumEvents from "ethereum-events";
+import {
+  BlockHistory,
+  bigNumberify,
+  cumulativeRewardScore,
+  handleERC20Transfer,
+} from "./utils";
+import {
+  ousdContract,
+  wousdContract,
+  OUSD_DEPLOY_BLOCK,
+  WOUSD_DEPLOY_BLOCK,
+} from "./contracts";
+import { ethereumEventsOptions, web3 } from "./config";
+
+const AIRDROP_AMOUNT = 400000000;
```

Comment:
Not sure if this is intentional or not, but this airdrop amount here is not in 1e18 fixed point.

This means that:
- the math calculations clip, rounding down to the nearest whole number.
- the CSV output in in non-fixed point, whole numbers only.
- 2/3s of the CSV output is "0,0,0" on the amounts.

Reply:
Pinging on this one.

Reply:
Updated this 👍